cluster-controller/src/main/java/io/strimzi/controller/cluster/operations/resource/AbstractOperations.java

```diff
@@ -87,17 +87,9 @@ public void create(T resource, Handler<AsyncResult<Void>> handler) {
                     }
                 },
                 false,
-                res -> {
-                    if (res.succeeded()) {
-                        log.info("{} {} has been created", resourceKind, resource);
-                        handler.handle(Future.succeededFuture());
-                    }
-                    else {
-                        log.error("{} creation failed:", resourceKind, res.cause());
-                        handler.handle(Future.failedFuture(res.cause()));
-                    }
-                }
```

Comment:
@tombentley without this, how we are logging that something went wrong ?

Reply:
@ppatierno the argument to the `blockingCodeHandler` parameter of the vertx `executeBlocking()` has a `try/catch` which already does logging. I can add a logging statement for the successful case.